test/tstRay.cpp

```diff
@@ -13,6 +13,176 @@
 
 #include <boost/test/unit_test.hpp>
 
+BOOST_AUTO_TEST_CASE(distance_box)
```

Comment:
Yes, I just copied what we have. If you don't want to have both I would rather keep the distance tests and remove the intersection tests since they test more.

Reply:
You mean keep the original one?

Reply:
No, the new checking distances. `inf` means "doesn't intersect", all other numbers mean "intersects". So we are checking more information.

Reply:
Oh right https://github.com/arborx/ArborX/blob/1e28e229b0c5bf0aaae8b908d5f7293cbebe5846/test/tstRay.cpp#L349 is the one that does the most.